internal/proto/encryptionkeyrotation.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package encryptionkeyrotation;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/proto";
+
+// EncKeyRotation holds the RPC method for running
+// key rotation operation on the volume.
+service EncKeyRotation {
+  rpc EncKeyRotate(EncKeyRotateRequest)
+  returns (EncKeyRotateResponse){}
+}
+// EncKeyRotateRequest contains the information needed to identify
+// the volume by the SP and access any backend services so that the key can be
```

Comment:
can you please reword this one as it wont be sent to SP rather its sent to the addons sidecar and it process the request